doc/sslang-lrm.tex

```diff
@@ -717,19 +717,27 @@ \subsection{Sequences}
 
 \subsection{Pattern Matching}
 \begin{grammar}
-<match> ::=  "match" <expr_blk> "{" (  <pat> "=>" "{" <expr> "}" )$^{+}$ "}"
+<match> ::=  "match" <expr_blk> "{" (  <pat> "=" "{" <expr> "}" )$^{+}$ "}"
 \end{grammar}
 
-Pattern matches are a succinct way to handle of a variety of cases for the expression in the match clause, known as the scrutinee. If the scrutinee satisfies the pattern in a match arm, the expression in the match arm is evaluated; otherwise, the expression in that arm is not evaluated.
+Pattern matches are used to break down algebraic data types.
+A pattern match consists of a ``scrutinee''---what is being matched on---and a number of ``arms.''
```

Comment:
nit: . outside of "

Reply:
Some will say that periods are supposed to be inside of sentences: https://www.grammar.com/periods-with-quotation-marks/

Though this seems to be due to reasons that are no longer relevant: https://style.mla.org/punctuation-and-quotation-marks/

I personally prefer what you suggest but am indifferent. @sedwards-lab do you have opinions here?

Reply:
The American standard rule is that the period be inside the quote. It's illogical from a math/CS perspective, but that's the convention here.

Reply:
haha ok nevermind then